tls/s2n_client_key_exchange.c

```diff
@@ -49,136 +67,91 @@ static int s2n_rsa_client_key_recv(struct s2n_connection *conn)
     client_protocol_version[1] = conn->client_protocol_version % 10;
 
     /* Decrypt the pre-master secret */
-    struct s2n_blob pms, encrypted;
-    pms.data = conn->secure.rsa_premaster_secret;
-    pms.size = S2N_TLS_SECRET_LEN;
+    struct s2n_blob encrypted;
```

Comment:
If you declare this slightly lower in the code, you can init it with the correct values.